rush-plugins/rush-azure-storage-build-cache-plugin/src/schemas/azure-interactive-auth.schema.json

```diff
@@ -4,21 +4,11 @@
 
   "type": "object",
 
-  "additionalProperties": false,
+  "additionalProperties": true,
 
   "required": ["storageAccountName", "storageContainerName"],
 
   "properties": {
-    "storageAccountName": {
```

Comment:
Removal of these properties doesn't make sense for the configuration of the build cache.